ekncontent/ekncontent/eknc-query-object.c

```diff
@@ -1072,3 +1006,160 @@ G_GNUC_END_IGNORE_DEPRECATIONS
     }
   return EKNC_QUERY_OBJECT (ret);
 }
+
+/*
+ * eknc_query_object_configure_enquire:
+ * @enquire: a #XapianEnquire object that will have properties set on it
+ *
+ * Private functon to configure a #XapianEnquire with the sorting parameters
+ * specified in @self.
+ */
+void
+eknc_query_object_configure_enquire (EkncQueryObject *self,
+                                     XapianEnquire *enquire)
+{
+  int sort_value = eknc_query_object_get_sort_value (self);
+
+  if (sort_value > -1)
+    {
+      EkncQueryObjectOrder order;
+      g_object_get (self, "order", &order, NULL);
```

Comment:
Why are you using `g_object_get()`, here?

Reply:
There wasn't an accessor method yet, though I could just as well make one.

Reply:
You can access `self->order` directly without the GValue boxing/unboxing around it